src/merkle/simple_smt/mod.rs

```diff
@@ -244,3 +258,13 @@ impl SimpleSmt {
         self.branches.insert(index, branch);
     }
 }
+
+// ITERATORS
+// ================================================================================================
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub struct SmtParentNode {
+    pub value: RpoDigest,
+    pub left: RpoDigest,
+    pub right: RpoDigest,
+}
```

Comment:
As mentioned in the other PRs - I'd move this into a central place. Regarding the name, we should probably settle on something that works across all structures. How about `InnerNodeInfo`?